lib/http.js

```diff
@@ -323,7 +323,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         req, expectedTarget, expectedRootCapability, expectedAction: 'read'
       });
 
-      const {index, equals, has} = req.body;
+      const {index, equals, has, count} = req.body;
```

Comment:
since there is no corresponding schema update, I guess we're not validating the body, do we have a meaningful tracking for getting validation schema done here?

Reply:
there are no validators on this endpoint. There is one for query strangely which is not used. I added `count` to the query validator here: https://github.com/digitalbazaar/bedrock-edv-storage/pull/38/commits/f8050aec6dd583c4ed9e66ffeadaa9b3776c28ba

Reply:
I added an issue for the validators: https://github.com/digitalbazaar/bedrock-edv-storage/issues/40

We are missing a few.